frontend/vite.config.js

```diff
@@ -0,0 +1,39 @@
+import vue from '@vitejs/plugin-vue'
+import pluginRewriteAll from 'vite-plugin-rewrite-all';
+import { visualizer } from "rollup-plugin-visualizer";
+import path from 'path'
+import { defineConfig } from 'vite'
```

Comment:
Ah good catch. I guess this file is not included in the formatter. I have made a note of this instead of just committing the suggestion since I noticed the semicolon is missing from line 1 as well.

Reply:
Fixed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1126/commits/70573c607524b18412be05649b4a90c520f50186